autopush/settings.py

```diff
@@ -124,9 +126,15 @@ def __init__(self,
         # CORS
         self.cors = enable_cors
 
+        # Force timeout in idle seconds
+        self.idle_timeout = idle_timeout
```

Comment:
Maybe `udp_timeout` is a better name? I'm worried this would cause confusion down the road, particularly if we add a handshake timeout, too.